codecov.yml

```diff
@@ -0,0 +1,11 @@
+coverage:
+  status:
+    project:
+      default:
+        enabled: no
```

Comment:
should this be 'yes' here and below?  Not seeing codecov in the checks section of the PR, though the coecove report is present.

Reply:
No, the intention is to remove codecov checks here since I don't think they are that useful